Documentation/ExtensionArchitecture/Tutorials/Tea/Model.rst

```diff
@@ -0,0 +1,298 @@
+:file:`ext_tables.sql`:
+
+..  include:: /CodeSnippets/Tutorials/Tea/ExtTablesSql.rst.txt
+
+The image is stored as an integer. However the field :sql:`image` in the
+database does not contains a reference to the image in form of an identifier.
+
+The field :sql:`image` stores how many images are attached. A separate table,
+a so called MM table stores the actual relationship. See about the definition
+of this field here: :ref:`extbase_tutorial_tea_model_columns_image`.
+
+..  hint::
+    If you have had a look at a TYPO3 database before you might wonder:
+    What about all those special fields like :sql:`uid`, :sql:`pid`,
+    :sql:`deleted` etc? The answer is TYPO3 will generate them automatically
+    for you.
+
+..  _extbase_tutorial_tea_model_tca:
+
+TCA - Table Configuration Array
+===============================
+
+The TCA makes the database model known to TYPO3. It defines all fields
+containing data and all semantic fields that have a special meaning within
+TYPO3 (like the :sql:`deleted` field used for soft deleting).
+
+TCA also defines how the input fields for the data in the backend should look
+like.
+
+The TCA is a nested PHP array. On the first level we need the the following
+keys in this example:
+
+:php:`ctrl`
+    Settings for the complete table, like an overall title, a lable that
+    should specify the single object, default sorting and the names of some
+    fields used internally.
+
+:php:`types`
+    We only have one type of tea record, however it is mandatory to
+    describe at least one type. Here we define the order in which
+    the fields are displayed in the backend.
+
+:php:`columns`
+    Here we define all fields that can be used for user input in the
+    backend.
+
+
+..  _extbase_tutorial_tea_model_ctrl:
+
+TCA :php:`ctrl` - Settings for the complete table
+-------------------------------------------------
+
+..  include:: /CodeSnippets/Tutorials/Tea/Configuration/TCA/TeaCtrl.rst.txt
+
+..  _extbase_tutorial_tea_model_ctrl_title:
+
+:php:`title`
+~~~~~~~~~~~~
+
+Defines the title used when we are talking about the table in the backend.
+It will be displayed on top of the list view of records in the backend
+and in backend forms.
+
+..  figure:: /Images/ManualScreenshots/ExtensionArchitecture/Tutorials/Tea/TeaTitle.png
+    :class: with-shadow
+
+    The **title** of the table tea
+
+Strings starting with :php:`LLL:` will be localized. See chapter
+:ref:`Extension localization <extension_localization>`. All other strings
+would be output as strings. You could also write:
+
+.. code-block:: php
+   :caption: EXT:tea/Configuration/TCA/tx_tea_domain_model_product_tea.php
+
+   [
+       'ctrl' => [
+           'title' => 'Tea',
+       ],
+   ]
+
+..  _extbase_tutorial_tea_model_ctrl_label:
+
+:php:`label`
+~~~~~~~~~~~~
+
+The :php:`label` is used as name for a specific tea record. The name is used
+in listings, dropdown fields used to reference a tea record and in the forms
+for editing records in the backend:
```

Comment:
This is shorter, but says about the same thing. Dropdowns are also lists, and they're also in backend forms.

```suggestion
The :php:`label` is used as name for a specific tea record. The name is used
in listings and in backend forms:
```